packages/schematics/angular/utility/latest-versions/package.json

```diff
@@ -11,10 +11,10 @@
     "karma-jasmine-html-reporter": "~1.7.0",
     "karma-jasmine": "~5.0.0",
     "karma": "~6.3.0",
-    "ng-packagr": "^14.0.0-next.2",
+    "ng-packagr": "^14.0.0-next.8",
     "rxjs": "~7.5.0",
     "tslib": "^2.3.0",
-    "typescript": "~4.6.2",
+    "typescript": "4.7.0-beta",
```

Comment:
This will cause new projects to be created with 4.7.0-beta.  Is that intended?

Reply:
I think this is fine since we have additional packages to use prereleases version. Prior to final release we should definitely make sure that this is update to a stable version.

Reply:
I've reverted it.